inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp

```diff
@@ -1825,7 +1825,8 @@ inline void MKLDNNReduceNode::reduce_kernel_process(const uint8_t *in_p, uint8_t
 }
 
 inline void MKLDNNReduceNode::reduce_kernel_post_process(uint8_t *out_ptr) {
-    const float divisor = static_cast<float>(IB * IC * ID * IH * IW / (OB * OC * OD * OH * OW));
+    const size_t divisor = IB * IC * ID * IH * IW / (OB * OC * OD * OH * OW);
+    const float casted_divisor = static_cast<float>(divisor);
```

Comment:
casted_divisor looks embarrassing. I would prefer to update the name of the temporary variable, bot major one.

Reply:
Temporary variable was renamed to `integerDivisor`, thanks